### ISSUE TITLE
Placing an alert

### ISSUE THREAD

**Issue by nicoloridulfo** [fhqvst/avanza#66]
Hi!
First of, great repository

It would be very convenient if the library had a function for placing an alert on a stock at a specific price and expiration. I could not find this in the source code. The app supports it, which means that there must be an end-point for it(?).

**Comment by fhqvst** [comment]
Yeah, that's a feature I've been wanting to have as well. Should definitely be possible to add - it's just a matter of setting up the proxy and seeing what API calls the app makes.

**Comment by nicoloridulfo** [comment]
It would be very useful!
I have always wondered how you were able to get the end-points (not an app reverse-engineer myself 😄)!

**Comment by nicoloridulfo** [comment]
I tried the proxy approach. Got pretty far, but got stuck with a certificate error after certificate error. Perhaps someone else has better luck.

**Comment by asifma** [comment]
Hey, couldn't agree more! Very useful repo 👍 
This would be very useful!! Anyone that wants to collaborate around this?

**Comment by fhqvst** [comment]
IIRC, there are also a couple other routes under `/_cqbe` that might be useful, such as fetching "inspiration lists" and more.

**Comment by asifma** [comment]
@fhqvst : Vill du kika på det ihop med mig? Om du har tid och lust :)

**Comment by asifma** [comment]
@fhqvst : Okey, I solved it. Will create a PR!

![image](https://user-images.githubusercontent.com/27285709/131254331-623b8559-becf-4ebd-aa4f-fcdf1c2a97bf.png)

**Comment by asifma** [comment]
PR Created, please review when you have time :) -> #68
